UnitedIncome/components
# Ignores gh-pages

### Description
> Please provide a brief description of your changes below.

Ignores the `gh-pages` branch from the Circle job as it will forever fail.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* `gh-pages` branch will no longer trigger CircleCI.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

* Reference material: https://circleci.com/docs/2.0/configuration-reference/#branches

### Comments
> Any additional comments that will aid with the review process.

I'd like to deprecate CircleCI from this workflow as some point soon. Likely part of the integration this will have to happen.
